spec/resource/status_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe Stormpath::Resource::Status, :vcr do
+describe 'Stormpath::Resource::Status', :vcr do
```

Comment:
This `Stormpath::Resource::Status` doesn't exist anymore after this change, right? We should rename this.